about.html

```diff
@@ -0,0 +1,105 @@
+      <li><a href="#">English</a></li>
+      <li><a href="#">My page</a></li>
+      <li><a href="#">logout</a></li>
+    </ul>
+    </div>
+  </div>
+  <header class="header">
+
+
+    <div class="menu">
+ 
+     <div id="menuBar"><img src="images/menu.png" 
+       alt="menubar-image" ></div>
+ 
+       <div class="logo"><a href="https://nessrine88.github.io/Capstone-Project/" target="_blank" rel="nofollow noopener"><img src="images/White And Black Modern Abstract Beauty Logo.png" alt="logo"></div></a>
+       <div id="menuList">
+        <ul>
+          <li><a href="./index.html" rel="nofollow noopener">Home</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#"><span class="sp">Best offers</span></a></li>
+        </ul>
+      </div>
+     </div>
+    
+   </header>
+
+   <section class="section1">
+    <h2 class="header2">"Take care of your beauty!"</h2>
+   <h1>Welcome to Sultan-Oils company</h1>
+   <p class="para1Section1">Immerse Yourself in Prickly Pear Wellness Retreat and Experience the Transformative Power of Nature. Reserve Your Spot Now to Indulge in Skincare, Wellness Workshops, Spa Treatments, and Community Connections. Join Us for a Memorable Journey Towards Radiant Skin and Holistic Well-being.</p>
+
+   <p class="para2Section1"> Our dedicated staff will be delighted to assist you with any inquiries or concerns you may have. To secure your spot or to learn more about the retreat, please feel free to contact our team at:<br><span class="sp1">infofigue@gmail.com</span></p>
+   </section>
+   <section class="section2">
+   <h2 class="S2header2">Sultan-Oils company</h2>
+   <div class="line"></div>
+   <p class="S2para1">We are here to provide you with the necessary information and support to ensure your participation in this remarkable event. Don't hesitate to reach out to us for any clarification or assistance you may need. </p>
+   <div class="logo"><a href="https://nessrine88.github.io/Capstone-Project/"> <img src="images/White And Black Modern Abstract Beauty Logo.png" alt="logo-image"></a></div>
+</section>
+
+<section class="section3">
+   <h2 class="S3header">Check out the previous reviews for our product.</h2>
+   <div class="line"></div>
+   <p class="S3para">Here are some of the reviews from our valued clients who have experienced our product firsthand.</p>
+   <div class="revieweImgs">
+    <div class="award img2019">
+      <div class="transparentbg">
+  <h2 class="S3h2">2019</h2>
+  <h3 class="S3h3"> "Celebrating the Victory of Emma Smith: Our Esteemed Winner of 2019".</h3>
+</div>
+</div>
+<div class="award img2021">
+  <div class="transparentbg">
+  <h2 class="S3h2">2021</h2>
+  <h3 class="S3h3">"Celebrating the Victory of Jamila henderson: Our Esteemed Winner of 2021".</h3>
+</div>
+</div>
+  </div>
+  </section>
```

Comment:
Kindly use the color guideline for the background color of this design in this section :

Your design :
![image](https://github.com/Nessrine88/Capstone-Project/assets/20537712/5d3c6179-5f5d-4eb7-996f-71b03d4fa1b4)

Design given :
![image](https://github.com/Nessrine88/Capstone-Project/assets/20537712/76814caf-f1a4-4337-bbaf-69d317f8c1e3)